<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>1035</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#scrollArea{
	background-color: qlineargradient(spread:pad, x1:0.023, y1:0.965909, x2:1, y2:0, stop:0 rgba(195, 195, 195, 255), stop:1 rgba(216, 216, 216, 255));
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius:12px;
}
#envTB
 {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}
#MainWindow
{
	/*background-color: qlineargradient(spread:pad, x1:0.023, y1:0.965909, x2:1, y2:0, stop:0 rgba(195, 195, 195, 255), stop:1 rgba(216, 216, 216, 255));*/
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>940</y>
      <width>178</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="renderPB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Render...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyPB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>489</width>
      <height>601</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>483</width>
       <height>595</height>
      </rect>
     </property>
     <widget class="QSpinBox" name="aoSB">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>60</y>
        <width>55</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Samples Per Pixel: </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sppSB">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>55</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="envTB">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>371</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="envCB">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Environment Map:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="aoCB">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Ambient Occlusion Samples: </string>
      </property>
     </widget>
     <widget class="QCheckBox" name="shadowsCB">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Shadows</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>421</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Camera</string>
      </property>
      <widget class="QDoubleSpinBox" name="focalDistanceSB">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>50</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Focal Distance:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Aperture:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="apertureSB">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>80</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
       <property name="value">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QRadioButton" name="pinholeRB">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>109</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Pinhole</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="thinLensRB">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>109</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>ThinLens</string>
       </property>
      </widget>
     </widget>
     <widget class="QToolButton" name="envPB">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>170</y>
        <width>26</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="skyDomeSB">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>230</y>
        <width>55</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="skyDomeCB">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Sky Dome</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Distance: </string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="ao_distanceSB">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>90</y>
        <width>62</width>
        <height>27</height>
       </rect>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>421</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Material</string>
      </property>
      <widget class="QRadioButton" name="phongRB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Phong</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="glassRB">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>109</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Glass</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="lambertianRB">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Lambertian</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="riSB">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>50</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
       <property name="value">
        <double>1.500000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ri :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ri 2:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="ri2SB">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>80</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QRadioButton" name="thinDielectricRB">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>ThinDielectric</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>80</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>thickness:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="thicknessSB">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>80</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>60</y>
         <width>31</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>eta :</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="etaSB">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>50</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
       <property name="value">
        <double>1.500000000000000</double>
       </property>
      </widget>
      <widget class="QRadioButton" name="transparentRB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Transparent</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="orenNayarRB">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>OrenNayar</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>140</y>
         <width>71</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>roughness:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="roughnessSB">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>160</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
       <property name="value">
        <double>1.500000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Reflectivity:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="reflectivitySB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>62</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="title">
     <string>Preview</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>890</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connecting...</string>
    </property>
   </widget>
   <widget class="GLWidget" name="glview" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>256</width>
      <height>256</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>256</width>
      <height>256</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="previewPB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>26</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="rotatePB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>26</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fpsLB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>256</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGLuRay">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuGLuRay"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
